modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices_{context}"]
+= Adding kernel arguments to enable IOMMU for the CPU
+
+To enable IOMMU extension for the BIOS associated with a PCI device, add kernel arguments to Intel or AMD devices. Create the `MachineConfig` object file to add the kernel arguments.
+
```

Comment:
I think we need to clarify this part a bit.

Virtualization and IOMMU extensions need to be enabled in the bios, as a prerequisite to this procedure. 
Similar to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/sect-troubleshooting-enabling_intel_vt_x_and_amd_v_virtualization_hardware_extensions_in_bios


Once these extensions are enabled in the BIOS only then the administrator should create the `MacnineConfig` to add the kernel argument that uses the IOMMU extension.

Reply:
I've updated the content. Please check the main introduction to this content and the prerequisites for the Adding kernel arguments... topic.